wgsi.py

```diff
@@ -20,20 +20,68 @@ class HttpResponse:
         __iter__(self): Permite tornar o objeto iterável
     """
 
-    def __init__(self, header, body, content_type, status) -> None:
-        self.header = header
-        self.body = body
-        self.content_type = content_type
+    def __init__(
+    self,
+    content="",
+    status="200 OK",
+    headers=None,
+    content_type="text/html;charset=UTF-8",
+    ):
+        self.content = content
         self.status = status
+        self.headers = headers or {}
+        self.headers["content-type"] = content_type
 
     def __iter__(self):
-        """
-        Permite o body ser interável
-        """
-        yield self.body.encode("utf-8")
-    
+        # iterável que será escrito no body do response
```

Comment:
`Tornar o content um objeto iterável, podendo percorre-lo com um for`

Reply:
faz sentido